### ISSUE TITLE
Signs 

### ISSUE THREAD

**Issue by DeadmanTheGod** [AbsolemJackdaw/AppleCrates#27]
Add editable signs to display with the crates. 
RGB support for the text would b nice.

**Comment by oliglance** [comment]
i personally don't see the need for this from a game design perspective, as the items display themselves already.
is there really a need for signs?

**Comment by AbsolemJackdaw** [comment]
The sign idea comes from a discontinued mod called Nekoration 
https://www.curseforge.com/minecraft/mc-mods/nekoration
check the images and see he signs. I think they could be fitting to pose next to crates to show and tell what you're selling


<img width="1920" height="1017" alt="Image" src="https://github.com/user-attachments/assets/5533f4a0-f9f7-4344-8eb3-c133e8919d88" />

**Comment by oliglance** [comment]
ohhh, a whole separate block, i see! would probably fit more in a separate mod, though? (looking at the current mod name "apple crates")

**Comment by AbsolemJackdaw** [comment]
given the shop statue of the mod, it could be a nice addition.
My vision of the mod turns around the shop-ability of medieval multiplayer based gameplay

<img width="854" height="480" alt="Image" src="https://github.com/user-attachments/assets/0584ba0f-a458-43f4-a382-cd445e1d1c28" />

**Comment by oliglance** [comment]
right, good point! whoever doesn't want the signs, in the end, could just make a data pack to disable the recipe, too.
